cumulus/polkadot-omni-node/lib/src/cli.rs

```diff
@@ -166,6 +166,14 @@ pub struct Cli<Config: CliConfig> {
 	#[arg(long)]
 	pub dev_block_time: Option<u64>,
 
+	/// Disable the statement store.
+	///
+	/// The statement store is an off-chain data-store for signed statements accessible via RPC
+	/// and OCW.
+	/// It uses the runtime api to get the allowance associated to an account.
+	#[arg(long)]
+	pub disable_statement_store: bool,
```

Comment:
As per the offline discussion, should we disable the statement store by default ?

Reply:
If we disable by default I fear nobody will enable it. But maybe it will be enough with the few that will enable it.

I feel going into a dedicated binary for people chain seems the better then no?

Reply:
I think that having a dedicated binary for the people chain would create more complications. We would need to release one more binary, change all the tests that involve the rococo-people chain to use the dedicated binary, change the deployment process only for the people chain. I don't know.

Or maybe we should just enable the statement-store by default as it is done here. I'm not sure which solution is the best. I'm curious what others think as well.

cc: @skunert @iulianbarbu @seemantaggarwal

Reply:
I have also this other PR https://github.com/paritytech/polkadot-sdk/pull/8421 that make the CLI configurable for the statement store. This way we wouldn't need to maintain a different service in a different binary. But it still requires a dedicated process for the release.

Reply:
I would go for support in the existing `polkadot-omni-node-lib` instead for a new binary too.

> If we disable by default I fear nobody will enable it.

Would it be widely useful for parachains? If no, only the ones requiring it will enable it, and that should be fine. I would rather go for the reverse logic here.